### PR TITLE
Initialize jxclient

### PR DESCRIPTION
**Pull request by fiunchinho**
- [X] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Description
We need to make sure we have a valid jx client, otherwise we may get a nil pointer dereference

### CODE REVIEW

File: pkg/jx/cmd/common_install.go [hunk 1]
```diff
@@ -1694,7 +1694,11 @@ func (o *CommonOptions) isProw() (bool, error) {
 	if ns == "" {
 		ns = o.currentNamespace
 	}
-	env, err := kube.GetEnvironment(o.jxClient, ns, "dev")
+	jxClient, _, err := o.JXClient()
```
Comment (ccojocar) [inline pkg/jx/cmd/common_install.go]: You can configure the jx client and set the dev environment in one call with `JXClientAndDevNamespace`.

**Comment by jstrachan** [comment]
@ccojocar I tweaked the PR to use `JXClientAndDevNamespace()`

File: pkg/jx/cmd/common_install.go (hunk 1 above)
Comment (jstrachan) [reply pkg/jx/cmd/common_install.go]: I've fixed this

**Comment by fiunchinho** [comment]
👍

**Comment by fiunchinho** [comment]
I guess someone needs to /ok_to_test ?

**Comment by pmuir** [comment]
/test this
Missing build log

**Pull request merged**